### ISSUE TITLE
Usage of HCLOUD_TOKEN environment variable does not work

### ISSUE THREAD

**Issue by cwilhelm** [vitobotta/hetzner-k3s#145]
If I use the command 
```
HCLOUD_TOKEN=mySecretToken hetzner-k3s create --config ./hetzner-k3s_cluster_config.yaml
```
with no hetzner_token entry in the hetzner-k3s_cluster_config.yaml file, I get the error 

```
Unhandled exception: Missing YAML attribute: hetzner_token at line 3, column 1 (YAML::ParseException)
  from /opt/homebrew/Cellar/hetzner_k3s/1.0.7/bin/hetzner-k3s in 'raise<YAML::ParseException>:NoReturn'
  from /opt/homebrew/Cellar/hetzner_k3s/1.0.7/bin/hetzner-k3s in 'YAML::Nodes::Node+@YAML::Nodes::Node#raise<String>:NoReturn'
  from /opt/homebrew/Cellar/hetzner_k3s/1.0.7/bin/hetzner-k3s in 'Configuration::Main@Object::from_yaml<String>:Configuration::Main'
  from /opt/homebrew/Cellar/hetzner_k3s/1.0.7/bin/hetzner-k3s in 'Configuration::Loader::new<String, Nil>:Configuration::Loader'
  from /opt/homebrew/Cellar/hetzner_k3s/1.0.7/bin/hetzner-k3s in 'Hetzner::K3s::CLI::Create#run:Nil'
  from /opt/homebrew/Cellar/hetzner_k3s/1.0.7/bin/hetzner-k3s in 'Hetzner::K3s::CLI#parse_and_run:Nil'
  from /opt/homebrew/Cellar/hetzner_k3s/1.0.7/bin/hetzner-k3s in '__crystal_main'
  from /opt/homebrew/Cellar/hetzner_k3s/1.0.7/bin/hetzner-k3s in 'main'
```

When I add the hetzner_token entry to hetzner-k3s_cluster_config.yaml file with the exact value, all is working.

@vitobotta thank you very much for creating this very useful tool.

**Comment by vitobotta** [comment]
Hi @cwilhelm - I recently rewrote the tool from Ruby to Crystal and it looks like I forgot to make the token in the config file optional if set in the env variable. I will fix this in the next release. I hope it's OK for now to keep the token in the config file (if you can't, can you use envsubst for now?)

**Comment by vitobotta** [comment]
@cwilhelm This is fixed in v1.0.8. Closing but feel free to reopen if needed.
